content/ಅಂಕೆಗಳ ಅಟ್ಟಳಿಗೆ.md

```diff
@@ -0,0 +1,134 @@
+
+`ಪತ್`-ಇಗೆ ವಿಭಕ್ತಿಸೂಚಕವಾದ ಅನುಸ್ವಾರ ಸಮೇತವಾದ ಪದಿಂ ಇತ್ಯಾದಿಗಳು ಆದೇಶವಾಗಲಾರದು [ಅನುಸ್ವಾರದ ಅನುಸಾರ ನೋಡಿ](https://github.com/amshuman-kr/kannadada-bhakti/blob/main/content/%E0%B2%B5%E0%B2%95%E0%B2%BE%E0%B2%B0%E0%B2%A6%20%E0%B2%B5%E0%B2%BF%E0%B2%AD%E0%B2%95%E0%B3%8D%E0%B2%A4%E0%B2%BF.md#%E0%B2%A6%E0%B3%8D%E0%B2%B5%E0%B2%BF%E0%B2%A4%E0%B3%8D%E0%B2%B5%E0%B2%B8%E0%B2%82%E0%B2%A7%E0%B2%BF). `ಪತಿಂ` ಅಥವಾ `ಪತ್ತಿಂ` ಗೆ ವಿಭಕ್ತರೂಪದ ಅಂದರೆ `ಪದಿಂ` -ನ ಆದೇಶವಾಗಿರಬಹುದು ಅಷ್ಟೇ.
+ಅದಲ್ಲದೇ, ಅನುಸ್ವಾರವು ಮಕಾರ ನಕಾರ ಗಳ ಉಚ್ಚಾರಣೆಗಳನ್ನು ಧರಿಸುವುದನ್ನು ಬೇರೆಡೆಯೂ [ನೋಡಿದ್ದೇವೆ](https://github.com/amshuman-kr/kannadada-bhakti/blob/main/content/%E0%B2%85%E0%B2%A8%E0%B3%81%E0%B2%B8%E0%B3%8D%E0%B2%B5%E0%B2%BE%E0%B2%B0%E0%B2%A6%20%E0%B2%85%E0%B2%A8%E0%B3%81%E0%B2%B8%E0%B2%BE%E0%B2%B0.md#%E0%B2%B9%E0%B2%B5%E0%B3%8D%E0%B2%AF%E0%B2%95%E0%B2%95%E0%B2%A8%E0%B3%8D%E0%B2%A8%E0%B2%A1%E0%B2%A6%E0%B2%B2%E0%B3%8D%E0%B2%B2%E0%B2%BF-%E0%B2%85%E0%B2%B0%E0%B3%8D%E0%B2%A7%E0%B2%BE%E0%B2%A8%E0%B3%81%E0%B2%B8%E0%B3%8D%E0%B2%B5%E0%B2%BE%E0%B2%B0-%E0%B2%B9%E0%B2%BE%E0%B2%97%E0%B3%82-%E0%B2%85%E0%B2%A6%E0%B3%81-%E0%B2%A8%E0%B2%95%E0%B2%BE%E0%B2%B0%E0%B2%B5%E0%B2%BE%E0%B2%97%E0%B3%81%E0%B2%B5-%E0%B2%AA%E0%B3%8D%E0%B2%B0%E0%B2%95%E0%B3%8D%E0%B2%B0%E0%B2%BF%E0%B2%AF%E0%B3%86). 
+
+## ಲಗಂ ವರ್ಜನೆ
+ಕನ್ನಡ ಭಾಷೆಯು ಶಬ್ದದಾರಂಭದಲ್ಲಿ ಲಗಂ ಜತಿಯನ್ನು ವರ್ಜಿಸುತ್ತದೆ. ಇದರ ಬಗ್ಗೆ [ಇಲ್ಲಿ](https://github.com/amshuman-kr/kannadada-bhakti/blob/main/content/%E0%B2%B5%E0%B2%95%E0%B2%BE%E0%B2%B0%E0%B2%A6%20%E0%B2%B5%E0%B2%BF%E0%B2%AD%E0%B2%95%E0%B3%8D%E0%B2%A4%E0%B2%BF.md#%E0%B2%A6%E0%B3%8D%E0%B2%B5%E0%B2%BF%E0%B2%A4%E0%B3%8D%E0%B2%B5%E0%B2%B8%E0%B2%82%E0%B2%A7%E0%B2%BF) ಓದಬಹುದು. ಜಗಣ ಅಥವಾ `ಲಗಂ` ವರ್ಜಿಸಲು ಕೆಲವು ಆಗಮ ಆದೇಶ ಲೋಪಗಳು ದ್ವಿತ್ವಗಳು ಉಂಟಾಗುತ್ತವೆ.
+
+# ಅಂಕಿಗಳ ಅಟ್ಟಣೆ
+1. ಕನ್ನಡದಲ್ಲಿ ಜಗಣ/ಲಗಂ ವರ್ಜ್ಯ
+2. ಹಳಗನ್ನಡದಲ್ಲಿ 10 = ಪತ್
+3. ಇಂ ಪ್ರತ್ಯಯ ಪಂಚಮೀ ವಿಭಕ್ತಿಯ ಸಾಪೇಕ್ಷ ದೂರವನ್ನು ಕೊಡುತ್ತದೆ
+4. ಅನುಸ್ವಾರವು ಮಕಾರ ನಕಾರ ಗಳೆರಡನ್ನೂ ಹೊಂದಬಲ್ಲುದು.
+ಈ ಮೇಲಿನ ಅಂಶಗಳನ್ನು ಮನದಲ್ಲಿಟ್ಟುಕೊಂಡು 10 - 18 ಅಂಕಿಗಳ ಅಟ್ಟಿಯನ್ನು ಮಾಡಬಹುದು.
+
+## ಹನ್ನೊಂದು
+ಪತ್ತಿಂ+ಒಂದು ->ಪದಿಂ+ಒಂದು -> ಪಯಿನ್+ಒಂದು -> ಪನ್+ಒಂದು -> ಪನೊಂದು -> ಪನ್ನೊಂದು (ಲಗಂ ವರ್ಜನೆ, ನಾನ ನಾನಾ ಜತಿಗಾಗಿ)
+
+## ಹನ್ನೆರಡು
+ಪತ್ತಿಂ+ಎರಡು ->ಪದಿಂ+ಎರಡು -> ಪಯಿನ್+ಎರಡು -> ಪನ್+ಎರಡು -> ಪನೆರಡು -> ಪನ್ನೆರಡು ( ನಾನ ನಾನಾ ಜತಿಗಾಗಿ)
+
+## ಹದಿಮೂರು
+ಪತ್ತಿಂ+ಮೂರು ->ಪದಿಂ+ಮೂರು -> ಪದಿಮ್+ಮೂರು -> ಪದಿಮ್ಮೂರು ->ಪದಿಮೂರು (ಲಗಂ ವರ್ಜನೆ)
+
+## ಹದಿನಾಲ್ಕು
+ಪತ್ತಿಂ+ನಾಲ್ಕು ->ಪದಿಂ+ನಾಲ್ಕು -> ಪದಿನ್+ನಾಲ್ಕು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನ್ನಾಲ್ಕು ->ಪದಿನಾಲ್ಕು (ಲಗಂ ವರ್ಜನೆ)
+
+## ಹದಿನೈದು
+ಪತ್ತಿಂ+ಐದು ->ಪದಿಂ+ಐದು -> ಪದಿನ್+ಐದು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನೈದು
+
+## ಹದಿನಾರು
+ಪತ್ತಿಂ+ಆರು ->ಪದಿಂ+ಆರು -> ಪದಿನ್+ಆರು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನಾರು
+
+## ಹದಿನೇಳು
+ಪತ್ತಿಂ+ಏಳು ->ಪದಿಂ+ಏಳು -> ಪದಿನ್+ಏಳು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನೇಳು
+
+## ಹದಿನೆಂಟು
+ಪತ್ತಿಂ+ಎಂಟು ->ಪದಿಂ+ಎಂಟು -> ಪದಿನ್+ಎಂಟು (ಅನುಸ್ವಾರದ ನಕಾರ ಉಚ್ಚಾರಣೆ) -> ಪದಿನೆಂಟು
+## ಹತ್ತೊಂಬತ್ತು +
+ಇಲ್ಲಿಂದ ಮುಂದೆ ಷಷ್ಠೀ ತತ್ಪುರುಷ ಸಮಾಸಗಳೇ ಕಾಣುತ್ತವೆ, ಪಂಚಮೀ ಇಲ್ಲ
+ಪತ್ತ+ಒಂಬತ್ತು -> ಪತ್ತೊಂಬತ್ತು
+ಪತ್ತರ+ಒಂಬತ್ತು -> ಪತ್ತ್ರೊಂಬತ್ತು ಉತ್ತರ ಕರ್ನಾಟಕದಲ್ಲಿ ಪ್ರಯೋಗದಲ್ಲಿದೆ
+ಇರ್+ಪತ್+ಅ +ಒಂದು -> ಇರ್ಪತ್ತೊಂದು -> ಇಪ್ಪತ್ತೊಂದು ರಲ್ಲಿ ಕೂಡಾ ಷಷ್ಠೀ ವಿಭಕ್ತಿಯ `ಅ` ಪ್ರತ್ಯಯ ಕಾಣುತ್ತದೆ.
```

Comment:
Mention that ಇಪ್ಪತ್ತ ಒಂದು is also used and is the original split form and ಇಪ್ಪತ್ತೊಂದು is not ಇಪ್ಪತ್ತು + ಒಂದು, citing ನೂರ ಎರಡು, ಸಾವಿರದ ಮೂರು etc.

Reply:
done